modules/rubiconBidAdapter.js

```diff
@@ -949,6 +868,81 @@ function addVideoParameters(data, bidRequest) {
   data.imp[0].video.h = size[1]
 }
 
+function applyFPD(bidRequest, mediaType, data) {
+  const bidFpd = {
+    user: {...bidRequest.params.visitor},
+    context: {...bidRequest.params.inventory}
```

Comment:
Sorry, upon reviewing this all again, I realize that putting rubicon-specific FPD on global objects is a data leakage mistake.

Please put:
- adunit/imp-level 'user' attributes in openrtb imp.ext.rubicon.visitor
- adunit/imp-level 'context' attributes in openrtb imp.ext.rubicon.inventory

Reply:
Made a test page and this all actually does work because it's a single-adunit request for a single bidder. Going to resolve these issues.